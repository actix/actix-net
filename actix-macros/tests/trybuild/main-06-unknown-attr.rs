#[actix_rt::main(foo = "bar")]
async fn async_main() {}

#[actix_rt::main(bar::baz)]
async fn async_main2() {}

fn main() {}
