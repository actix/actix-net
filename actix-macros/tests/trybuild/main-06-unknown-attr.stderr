error: Unknown attribute specified
 --> $DIR/main-06-unknown-attr.rs:1:18
  |
1 | #[actix_rt::main(foo = "bar")]
  |                  ^^^^^^^^^^^

error: Unknown attribute specified
 --> $DIR/main-06-unknown-attr.rs:4:18
  |
4 | #[actix_rt::main(bar::baz)]
  |                  ^^^^^^^^
